test/rateLimiters/tokenBucket.test.ts

```diff
@@ -1,28 +1,29 @@
-import redis from 'redis-mock';
-import { RedisClientType } from 'redis';
+import { Redis as RedisType } from 'ioredis';
 import TokenBucket from '../../src/rateLimiters/tokenBucket';
 
+const RedisMock = require('ioredis-mock');
```

Comment:
The build is failing because of this require statement(typescript-esint no-var-requires). The tests only work with `ioredis-mock` setup like this. Working on a solution.